rest/snippets/UserRecordings/RecordingGroupEntity.yml

```diff
@@ -39,3 +39,122 @@ RecordingGroupEntity:
               type: string
               description: The ID of the series resource
               example: "abdf8785-cb23-445b-946f-69e8770500a1"
+
+RecordingEntity:
```

Comment:
Lets not have duplicated code, try to add `is_stream_available` to `RecordingEntity.yml `
and check if it is possible to make it available only for grouped/ungrouped endpoints